weblab_cg/data/tests/weblab_model.pyx

```diff
@@ -173,17 +173,17 @@ cdef class TestModel(CvodeSolver):
         self.initialState[2] = 0.6
         self.initialState[3] = 0.325
 
-        # Mapping of parameter oxmeta names to parameter array indices
+        # Mapping of parameter qualified names to parameter array indices
         self.parameterMap = {}
-        self.parameterMap['membrane_fast_sodium_current_conductance'] = 0
-        self.parameterMap['membrane_potassium_current_conductance'] = 1
+        self.parameterMap['https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#membrane_fast_sodium_current_conductance'] = 0
```

Comment:
Should be just local names here I think?

Reply:
Done!